sequencers/teiid-modeshape-sequencer-vdb/src/main/java/org/teiid/modeshape/sequencer/vdb/VdbExporter.java

```diff
@@ -202,6 +202,22 @@ private VdbManifest constructManifest( final Node vdb,
                                         source.setJndiName( jndiName );
                                     }
 
+                                    /*
+                                     * If the source node has an origin connection property then it denotes
+                                     * the connection that the model source is based on. This has to be preserved
+                                     * in the parent model properties.
+                                     */
+                                    if (sourceNode.hasProperty(VdbLexicon.Source.ORIGIN_CONNECTION)) {
+                                        String connName = sourceNode.getProperty(VdbLexicon.Source.ORIGIN_CONNECTION).getString();
+                                        if (connName != null) {
+                                            /*
+                                             * Add a property to the model that specifies the association
+                                             * between the named source and the named connection
+                                             */
+                                            model.getProperties().put(VdbModel.ORIGIN_CONNECTION_PROPERTY + "-" + source.getName(), connName);
```

Comment:
This the way the prefix is constructed, `VdbModel.ORIGIN_CONNECTION_PROPERTY + "-"`, is done 3 times in 3 different files. I think another constant in `VdbModel` would be appropriate. Wdyt?

Reply:
Fixed.